libs/services/designer-client-services/src/lib/base/operationmanifest.ts

```diff
@@ -43,6 +43,7 @@ import { ArgumentException, equals, UnsupportedException } from '@microsoft/util
 import type { OperationInfo, OperationManifest } from '@microsoft/utils-logic-apps';
 
 const apimanagement = 'apimanagement';
+const apimanagementtrigger = 'apimanagementtrigger';
```

Comment:
Anything related to consumption like any trigger/action specific to consumption should be present in consumption service, sendtobatch is an actionType and not a triggertype.. , batch is the trigger type. Your changes below breaks the batch trigger in standard.

YOur changes in getOPerationInfo assumes sendtobatch as a operation type which is common for both trigger and action. Please do test out the changes and if its not complete please do not checkin